HW_3/hw_task_2.py

```diff
@@ -0,0 +1,38 @@
+# В большой текстовой строке подсчитать количество встречаемых слов
+# и вернуть 10 самых частых. Не учитывать знаки препинания и регистр символов.
+# За основу возьмите любую статью из википедии или из документации к языку.
+import re
+import string
+
+text = "Гостеприимно распахнуло двери перед своими будущими учениками в 1992 году Краснодарское хореографическое училище – " \
+       "первое среднее профессиональное образовательное учреждение в области хореографического искусства на Северном Кавказе. " \
+       "130 учащихся и 50  педагогов объединены одной общей целью: создание атмосферы творческого труда, взаимопонимания, " \
+       "высокого профессионального мастерства. Ведь каждый день жизни училища – это огромная, кропотливая до пота работа " \
+       "его педагогов. Итог этой работы ежегодно виден на отчетном концерте училища. Краснодарское хореографическое училище " \
+       "дает высококачественное профессиональное образование через исторически сложившуюся и значимую для культуры " \
+       "систему хореографического искусства, готовя специалистов по двум направлениям: «Артист балета, преподаватель», " \
+       "«Артист балета ансамбля песни и танца танцевального коллектива, преподаватель». Сегодня училище вышло на новый " \
+       "уровень развития благодаря огромной поддержке министерства культуры Краснодарского края, совместной работе " \
+       "с руководством ведущих театрально-концертных учреждений г. Краснодара: Краснодарское творческое объединение " \
+       "«Премьера» им. Л.Г. Гатова, Краснодарская филармония имени Г.Ф. Пономаренко, Кубанский казачий хор, " \
+       "Краснодарское творческое объединение «Премьера». Уже стало традиционным, что обучающиеся училища выступают на " \
+       "одной сцене с лучшими артистами балета. Училище еще молодое, но устремлено в будущее, открыто всему новому, " \
+       "ведь оно – частичка огромного сердца Кубани, которая живет с ней едиными стремительными современными ритмами."
+
+
+NUM_WORDS = 10
+
+new_text = text.replace('–', '').lower().split()
+
+for i in range(len(new_text)):
+    new_text[i] = new_text[i].strip('.,-"/*!?;:«»{}()')
```

Comment:
Используйте переменную `punctuation` из модуля `string`, чтобы получить все символы пунктуации, для создания текста без этих символов

Reply:
Например так:
```suggestion
from string import punctuation 
new_text = ''.join(char for char in text.lower() if char not in string.punctuation)
```
или так 
```suggestion
new_text = ''.join(filter(lambda x: x not in string.punctuation, text.lower()))
```